e2e/testsuite/testsuite.go

```diff
@@ -60,6 +60,7 @@ type E2ETestSuite struct {
 
 	grpcClients    map[string]GRPCClients
 	paths          map[string]path
+	relayers       map[ibc.Wallet]bool
```

Comment:
while we do currently just run a single test in each run, we should try and make sure that if we change that we need to make as few changes as possible. With relayers being at the top level, this means that these relayer wallets would be shared across all tests. What might make sense to do is to create a mapping of testName -> relayer pair. Similar to how we do [here](https://github.com/cosmos/ibc-go/blob/9bca644453430b5daab860cc6e3bb1fd3b45245d/e2e/testsuite/testsuite.go#L250).

This could even be a `map[string]map[ibc.Wallet]bool` which isn't super pretty, but maybe we could work with an alias of some sort here.

Reply:
I did go with an alias here and basically just added a couple of functions on it. Lmk what you think.